doc/lua_api.txt

```diff
@@ -3664,7 +3664,9 @@ Definition tables
         ^ Don't forget to use "leveled" type nodebox. ]]
         liquid_range = 8, -- number of flowing nodes around source (max. 8)
         drowning = 0, -- Player will take this amount of damage if no bubbles are left
-        light_source = 0, -- Amount of light emitted by node (max. 14)
+        light_source = 0, --[[
+        ^ Amount of light emitted by node (max. 14).
+        ^ A value of 15 causes a lighting bug. ]]
```

Comment:
Fair enough, but it should still be made known that core.LIGHT_MAX is the constant to use for maximum light.  We certainly don't want modders to get in the habit of hard coding numeric literals for things that might change in a future version.